standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CompactionTxnHandler.java

```diff
@@ -202,16 +202,18 @@ public Set<CompactionInfo> findPotentialCompactions(int abortedThreshold,
    * This will grab the next compaction request off of
    * the queue, and assign it to the worker.
    * @param workerId id of the worker calling this, will be recorded in the db
+   * @param poolName id of the worker calling this, will be recorded in the db
    * @deprecated  Replaced by
    *     {@link CompactionTxnHandler#findNextToCompact(org.apache.hadoop.hive.metastore.api.FindNextCompactRequest)}
    * @return an info element for next compaction in the queue, or null if there is no work to do now.
    */
   @Deprecated
```

Comment:
why it's deprecated?

Reply:
It's kept for backward compatibility reasons. 
`public CompactionInfo findNextToCompact(FindNextCompactRequest rqst) throws MetaException`
should be used instead.